guides/deployments/podman.md

```diff
@@ -14,6 +14,18 @@ Linux kernel doesn't support CVMs.
 > Please be aware that there are
 > [limitations related to running Podman in rootless mode](https://github.com/containers/podman/blob/main/rootless.md#shortcomings-of-rootless-podman).
 
+> **Note**: Running Podman in rootless mode requires a FUSE device to implement
```

Comment:
the linter doesn't like having two adjacent blockquote sections. We could make this top-level text or combine it with the above (since these are really limitations of the approach). I just wanted to describe in greater depth why this method is incompatible with SELinux/AppArmor